testing/jormungandr-integration-tests/src/common/jcli/command/transaction.rs

```diff
@@ -186,6 +186,30 @@ impl TransactionCommand {
         self
     }
 
+    pub fn make_transaction(
+        mut self,
+        host: String,
+        sender: jormungandr_lib::interfaces::Address,
+        value: jormungandr_lib::interfaces::Value,
+        block0_hash: String,
+        secret: impl AsRef<Path>,
+        staging_file: impl AsRef<Path>,
```

Comment:
Uhm, but who is the receiver?

Reply:
The receiver is a testing account. Actually yesterday I was thinking that I may have missed making it optional. So if no address is provided a testing account is used.